assets/component-facets.css

```diff
@@ -75,6 +75,21 @@
   width: auto;
 }
 
+.facet-filters__field .select:after,
+.facet-filters__field .select:before,
+.mobile-facets__sort .select:after, 
+.mobile-facets__sort .select:before {
+  content: none;
+}
+
+.facet-filters__field .select__select,
+.mobile-facets__sort .select__select {
+  border-radius: 0;
+  min-width: auto;
+  min-height: auto;
+  transition: none;
+}
+
```

Comment:
Should we leave the focus style on it when we click it?
[video](https://screenshot.click/22-01-c6lrm-rpg47.mp4)

Reply:
Yes, this is expected!